utils/make_bitbake_license_list.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2022 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+if [[ "$1" =~ "-.*" ]]; then
+    echo "Usage: $(dirname "$0") <PREFIX>" 1>&2
+    exit 1
+fi
+
+for license in $(grep '^[0-9a-f]' LIC_FILES_CHKSUM.sha256 | awk '{print $2}'); do
+    md5sum="$(md5sum "$license" | awk '{print $1}')"
+    echo "    file://${1#file://}$license;md5=$md5sum \\"
+done
```

Comment:
This script I really didn't understand... But I guess I'll see it in action soon in another PR :)

Reply:
Yes, you will! 🙂